tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/discord/DiscordWebHookProcessor.java

```diff
@@ -84,6 +86,16 @@ public boolean sendDiscordWebHook(String webHookURL, String discordWebHookPayloa
             httpPost.addHeader("Accept-Language", "en-US,en;q=0.5");
             httpPost.addHeader("Content-Type", "application/json");
             httpPost.setEntity(new StringEntity(discordWebHookPayload, HTTP_CHARSET));
+            String httpProxyHost = System.getProperty("http.proxyHost");
+            String httpProxyPort = System.getProperty("http.proxyPort");
+            if (httpProxyHost!=null && httpProxyPort.trim().length()>0 && httpProxyPort!=null) {
+                Integer port = Integer.valueOf(httpProxyPort);
```

Comment:
Is there a reason that that the wrapper instead of primitive type is used here?
If not, this could be replaced with:
```java
int port = Integer.parseInt(httpProxyPort);
```